README.md

```diff
@@ -6,17 +6,25 @@ Homebrew formula for neovim.
 
 ## Usage
 
-```text
-$ brew tap neovim/neovim
-$ brew install --HEAD neovim
-```
+To install the latest release:
+
+    $ brew tap neovim/neovim
+    $ brew install neovim
+
+To install from the latest GitHub revision:
+
+    $ brew tap neovim/neovim
+    $ brew install --HEAD neovim
 
 To **upgrade** from a previous version:
 
-```text
-$ brew update
-$ brew reinstall --HEAD neovim
-```
+    $ brew update
+    $ brew reinstall neovim
```

Comment:
If there is stable version, it should use `brew upgrade neovim` to perform upgrade.